samples/ReverseProxy.HttpSysDelegation.Sample/ReverseProxy/appsettings.json

```
@@ -0,0 +1,45 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft": "Warning",
+      "Microsoft.Hosting.Lifetime": "Information"
+    }
+  },
+  "AllowedHosts": "*",
+  "ReverseProxy": {
+    "Routes": {
+      "delegateroute": {
+        "ClusterId": "delegatecluster",
+        "Match": {
+          "Path": "/delegate/{**catch-all}"
+        }
+      },
+      "proxyroute": {
+        "ClusterId": "proxycluster",
+        "Match": {
+          "Path": "{**catch-all}"
+        }
+      }
+    },
+    "Clusters": {
+      "delegatecluster": {
+        "Destinations": {
+          "SampleHttpSysServer": {
+            "Address": "http://localhost:5600/",
+            "Metadata": {
+              "HttpSysQueueName": "SampleHttpSysServerQueue"
```

`HttpSysDelegation` is used elsewhere to identify the same feature.
```suggestion
              "HttpSysDelegationQueue": "SampleHttpSysServerQueue"
```